### PR TITLE
Add gibbs module

### PR DESCRIPTION
**Pull request by arnaudbore**

### CODE REVIEW

File: modules/nf-scil/preproc/gibbs/main.nf [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+    output:
+    tuple val(meta), path("*dwi_gibbs_corrected.nii.gz")     , emit: dwi
+    path "versions.yml"                         , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
```
Comment (Manonedde) [inline modules/nf-scil/preproc/gibbs/main.nf:20]: you don't have any args, you can delete this line

File: modules/nf-scil/preproc/gibbs/main.nf [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    mrdegibbs $dwi ${prefix}__dwi_gibbs_corrected.nii.gz -nthreads 1
+
+    cat <<-END_VERSIONS > versions.yml
+    "${task.process}":
+        mrtrix: \$(mrdegibbs -version 2>&1 | sed -n 's/== mrdegibbs \\([0-9.]\\+\\).*/\\1/p')
+    END_VERSIONS
+    """
+
+    stub:
+    def args = task.ext.args ?: ''
```
Comment (Manonedde) [inline modules/nf-scil/preproc/gibbs/main.nf:35]: Same as previous comment

**Review by Manonedde — CHANGES_REQUESTED** [review]
Minor comments, looks good to me

**Review by Manonedde — APPROVED** [review]

**Review by AlexVCaron — APPROVED** [review]
All good 🚀 LGTM

**Pull request merged**